pairing/src/lib.rs

```diff
@@ -23,14 +23,13 @@ pub mod bls12_381;
 use core::ops::Mul;
 use ff::{Field, PrimeField, ScalarEngine};
```

Comment:
The lint here is expected; it will be resolved when both this PR and #237 have been merged.